### PR TITLE
Add support for auto-detecting single-stack IPV4/IPV6 Openshift clusters.

### PR DESCRIPTION
**Pull request by timflannagan1**
Dockerfile(s): Add the python `netaddr` module to allow the usage of the `ipv4` Jinja2 filter.

Charts/: This would add a top-level variable that enables IPV6 networking to values.yaml.

Images/: Update the MeteringConfig Ansible role to allow support for inspecting the Networking config.openshift.io object, iterating over the serviceNetwork array and checking if there are any IPV4 addresses. In the case where there's a mixture of IPV4 and IPV6 addresses, treat that configuration as a single-stack IPV4 Openshift cluster. Else, treat it as a single-stack IPV6 Openshift cluster.

### CODE REVIEW

**Comment by timflannagan1** [comment]
I realize I only tested this in isolation and that the metering-operator service account does not permissions to view cluster-wide resources.
/hold

**Comment by timflannagan1** [comment]
Updated the charts/manifests to allow the metering-operator service account the requisite permissions to view the cluster's networking configuration.
/hold cancel

File: charts/metering-ansible-operator/values.yaml [hunk 1]
```diff
@@ -36,6 +36,13 @@ operator:
       verbs:
       - get
       - list
+    # grants access to view the networking config
```
Comment (bentito) [inline charts/metering-ansible-operator/values.yaml]: oof, so many places to add the exact same lines.

Comment (timflannagan1) [reply charts/metering-ansible-operator/values.yaml]: I know, we manage too many sets of manifest files.

**Pull request merged**